apis/v1alpha1/generator.yaml

```diff
@@ -173,3 +182,26 @@ resources:
       ignore: true
     update_operation:
       custom_method_name: customUpdateHostedZone
+  HealthCheck:
+    renames:
+      operations:
+        GetHealthCheck:
+          input_fields:
+            HealthCheckId: Id
+    exceptions:
+      terminal_codes:
+      - TooManyHealthChecks
+      - HealthCheckAlreadyExists
+      - InvalidInput
+      - HealthCheckInUse
+    fields:
+      Tags:
+        from:
+          operation: ChangeTagsForResource
+          path: AddTags
+        compare:
+          is_ignored: True
```

Comment:
nit: I think right now the code-generator outputs code that can compare tags.

Reply:
Ok, I removed those lines.